src/WingetCreateCLI/Commands/NewCommand.cs

```diff
@@ -1,142 +1,142 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license.
-
-namespace Microsoft.WingetCreateCLI.Commands
-{
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+
+namespace Microsoft.WingetCreateCLI.Commands

```

Comment:
Revert this file, only whitespace changed

Reply:
reverted this file